addons/pos_restaurant/static/src/app/split_bill_screen/split_bill_screen.js

```diff
@@ -29,6 +29,10 @@ export class SplitBillScreen extends Component {
         return Object.values(this.priceTracker).reduce((a, b) => a + b, 0);
     }
 
+    get numberOfProducts() {
+        return Object.values(this.qtyTracker).reduce((a, b) => a + b, 0);
+    }
```

Comment:
here it's a matter of preference, but if you use `get` it's a bit unclear when you read the code whether you are looking at a property or a method.